spec/builders/subscriber_auth_email_builder_spec.rb

```diff
@@ -18,10 +18,30 @@
       expect { call }.to change(Email, :count).by(1)
     end
 
-    it "has a link to authenticate" do
-      link = "http://www.dev.gov.uk/destination?token=secret"
+    it "has a subject line prompting the user to manage their subscriptions" do
+      subject = "Manage your GOV.UK email subscriptions"
       email = call
-      expect(email.body).to include(link)
+      expect(email.subject).to include(subject)
+    end
+
+    it "has body content describing how to unsubscribe or manage subscriptions" do
+      line_one = "Manage your GOV.UK email subscriptions"
+      line_two = "You can unsubscribe from emails or change your subscription at:"
+      line_three = "http://www.dev.gov.uk/destination?token=secret"
+      line_four = "You’ll need to confirm your email address before you can make any changes."
+      line_five = "The link will stop working in 7 days."
+      line_six = "Didn’t request this email?"
+      line_seven = "Ignore or delete this email if you didn’t request it. Your subscriptions won’t be changed."
+      line_eight = "[Contact GOV.UK](https://www.gov.uk/contact/govuk) if you have any problems with your email subscriptions."
+      email = call
+      expect(email.body).to include(line_one)
+      expect(email.body).to include(line_two)
+      expect(email.body).to include(line_three)
+      expect(email.body).to include(line_four)
+      expect(email.body).to include(line_five)
+      expect(email.body).to include(line_six)
+      expect(email.body).to include(line_seven)
+      expect(email.body).to include(line_eight)
```

Comment:
How come this is multiple expectations?

Reply:
yeah... i know it's a bit ugly. Had some difficulty with newlines and whitespace matching if I put the fixture in as one string the right level of indentation.

Reply:
is there a better way to check a block of text with line breaks?

Reply:
hmmm does this work?
```ruby
it "has body content describing how to unsubscribe or manage subscriptions" do
    <<~BODY
       # Click the link to confirm your email address
       # Manage your GOV.UK email subscriptions

       ^ [Confirm your email address](#{link})
       You can unsubscribe from emails or change your subscription at:

       You need to do this to manage your GOV.UK email subscriptions. The link will stop working in 7 days.
       #{link}

       You’ll need to confirm your email address before you can make any changes.

       The link will stop working in 7 days.

       # Didn’t request this email?

      Ignore or delete this email if you didn’t request it. Your subscriptions won’t be changed.
      [Contact GOV.UK](https://www.gov.uk/contact/govuk) if you have any problems with your email subscriptions.
    BODY
    email.call
    expect(email.body).to eq(BODY)
end
```

Reply:
Although that feel like a pointless test

Reply:
I don't mind either way - maybe just checking for the link would suffice?

Reply:
@huwd I agree with @theseanything , I'm not sure it's worth checking every single line. I think just checking for the link is ok.

Reply:
@vanitabarrett cool cool, I think initially there was more of a switch we had two types of email copy i wanted to test. But yeah, now it's simpler, i should have simplified this too!

Reply:
that's amended